.env.example

```
@@ -0,0 +1,44 @@
+VITE_USE_POUCHDB=true
+
+# PouchDB Configuration (only used when VITE_USE_POUCHDB=true)
+# Local database name
+VITE_POUCHDB_LOCAL_NAME=fintrac
+
+# Remote CouchDB URL for sync
+VITE_COUCHDB_URL=http://localhost:5984/fintrac
+
+# CouchDB Authentication
+VITE_COUCHDB_USERNAME=admin
+VITE_COUCHDB_PASSWORD=password
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_COUCHDB_PASSWORD key should go before the VITE_COUCHDB_USERNAME key

(UnorderedKey)